Maths/MatrixAddition.java

```diff
@@ -0,0 +1,38 @@
+package Maths;
+
+/**
+ * https://en.wikipedia.org/wiki/Matrix_addition
+ */
+public class MatrixAddition {
+    public static void main(String[] args) {
+        int[][] a = new int[][] {{1, 2, 3}, {1, 2, 3}};
+        int[][] b = new int[][] {{0, 0, 0}, {1, 1, 1}};
+        int[][] c = new int[][] {{1, 2, 3}, {2, 3, 4}};
+
+        /* test addition */
+        assert addMatrices(a, b) == c;
```

Comment:
If you want to add test please create new unit test class. Please don't add them over here. Or you can remove test part completely. 

Please do needful.